client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -386,7 +386,7 @@ async function serialize(
             if (meta.type === 'image-sequence') {
               column2 = meta.originalImageFiles[feature.frame];
             } else if (meta.type === 'video') {
-              column2 = moment.utc(feature.frame / meta.fps).format('H:M:S.SSS');
+              column2 = moment.utc(feature.frame / meta.fps).format('HH:mm:ss.SSS');
```

Comment:
Notice the terrifying bug here.   `M` instead of `m`  indicates that until now, this was outputting the MONTH instead of the minute.